java/cuvs-java/src/test/java/com/nvidia/cuvs/CagraBuildAndSearchIT.java

```diff
@@ -200,272 +236,264 @@ public void testPrefilteringReducesResults() throws Throwable {
-                .withCagraGraphBuildAlgo(CagraGraphBuildAlgo.NN_DESCENT)
-                .withGraphDegree(2)
-                .withIntermediateGraphDegree(4)
-                .withNumWriterThreads(4)
-                .withMetric(CuvsDistanceType.L2Expanded)
-                .build();
-
-            CagraMergeParams physicalMergeParams = new CagraMergeParams.Builder()
-                .withOutputIndexParams(outputIndexParams)
-                .withStrategy(MergeStrategy.PHYSICAL)
-                .build();
-
-            log.info("Merging indexes with PHYSICAL strategy...");
-            CagraIndex physicalMergedIndex = CagraIndex.merge(new CagraIndex[]{index1, index2}, physicalMergeParams);
-            log.info("Physical merge completed successfully");
-
-            CagraSearchParams searchParams = new CagraSearchParams.Builder(resources)
-                .build();
-
-            List<Integer> mergedMap = Arrays.asList(0, 1, 2, 3);
-            CagraQuery query = new CagraQuery.Builder()
-                .withTopK(3)
-                .withSearchParams(searchParams)
-                .withQueryVectors(queries)
-                .withMapping(mergedMap)
-                .build();
-
-            log.info("Searching physically merged index...");
-            SearchResults physicalResults = physicalMergedIndex.search(query);
-            assertNotNull("Physical merge search results should not be null", physicalResults);
-            assertEquals("Physical merge search results should match expected", expectedResults, physicalResults.getResults());
-
-            // --- Serialization/deserialization check for both merged indexes ---
-            String physicalIndexFileName = UUID.randomUUID().toString() + ".cag";
-            physicalMergedIndex.serialize(new FileOutputStream(physicalIndexFileName));
-
-            File physicalIndexFile = new File(physicalIndexFileName);
-            InputStream physicalInputStream = new FileInputStream(physicalIndexFile);
-            CagraIndex loadedPhysicalIndex = CagraIndex.newBuilder(resources)
-                .from(physicalInputStream)
-                .build();
-
-            SearchResults resultsFromLoadedPhysical = loadedPhysicalIndex.search(query);
-            assertEquals("Loaded physical index search results should match expected",
-                expectedResults, resultsFromLoadedPhysical.getResults());
-
-            if (physicalIndexFile.exists()) {
-                physicalIndexFile.delete();
-            }
-            index1.destroyIndex();
-            index2.destroyIndex();
-            physicalMergedIndex.destroyIndex();
-            loadedPhysicalIndex.destroyIndex();
-        }
     }
+  }
 
+  // Commented out test for Logical merge strategy as it is not yet implemented in C yet
+  @Test
+  public void testMergeStrategies() throws Throwable {
+    float[][] vector1 = {
+      {0.0f, 0.0f},
+      {1.0f, 1.0f}
+    };
+
+    float[][] vector2 = {
```

Comment:
Regarding `testMergeStrategies` and `testMergingIndexes`:  Am I right in understanding that these are only whitespace changes?

Reply:
Yeah, sorry about that, I tried to use the same convention but the file already had mixed 2 and 4 spaces indents. BTW, is there a convention?